Composer/packages/server/src/models/asset/assetManager.ts

```diff
@@ -110,30 +108,9 @@ export class AssetManager {
       log('About to create folder', dstDir);
       mkdirSync(dstDir, { recursive: true });
 
-      // find selected template
       const npmPackageName = templateId === QnABotTemplateId ? 'generator-empty-bot' : templateId;
-      const generatorName = npmPackageName.toLowerCase().replace('generator-', '');
-
-      // create yeoman environment
-      const yeomanEnv = yeoman.createEnv(
-        '',
-        { yeomanRepository: templateGeneratorPath },
-        new TerminalAdapter({ console: console })
-      );
-      yeomanEnv.lookupLocalPackages();
-
-      const remoteTemplateAvailable = await this.installRemoteTemplate(
-        yeomanEnv,
-        generatorName,
-        npmPackageName,
-        templateVersion
-      );
-
-      if (remoteTemplateAvailable) {
-        await this.instantiateRemoteTemplate(yeomanEnv, generatorName, dstDir, projectName);
-      } else {
-        throw new Error(`error hit when installing remote template`);
-      }
+
+      await startYeomanTemplateWork(npmPackageName, templateVersion, dstDir, projectName, jobId);
 
       ref.path = `${ref.path}/${projectName}`;
```

Comment:
when this gets merged, let's make sure we don't revert the change needed for multibots, seen here:

https://github.com/microsoft/BotFramework-Composer/pull/6250/files#diff-bab4ce8cda0d657cc0bb0d75fc68eaea1d94673aed6b0ec3a807b50284d45101L138

Reply:
Sounds good, will ensure that that when I handle merge conflicts

Reply:
no merge conflicts, the bit of code you outlined is safe and sound in the merge output